repos/fdbt-site/src/pages/api/fareType.ts

```diff
@@ -17,15 +22,16 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
             const services = await getAllServicesByNocCode(nocCode);
             const hasBodsServices = services.some((service) => service.dataSource && service.dataSource === 'bods');
             // removed as TNDS is being disabled until further notice
-            // const hasTndsServices = services.some((service) => service.dataSource && service.dataSource === 'tnds');
+            //const hasTndsServices = services.some((service) => service.dataSource && service.dataSource === 'tnds');
+            const modes = getSessionAttribute(req, MULTI_MODAL_ATTRIBUTE);
```

Comment:
```suggestion
            const multiModalAttribute = getSessionAttribute(req, MULTI_MODAL_ATTRIBUTE);
```

Reply:
if we call this modes, and we wanted to get the modes out of it, we would have to do `modes.modes` which is obviously not great.